webrtc/RTCPeerConnection-getDefaultIceServers.html

```diff
@@ -35,7 +35,6 @@
    */
 
   test(() => {
-    assert_own_property(RTCPeerConnection, 'getDefaultIceServers');
```

Comment:
Now you're deleting this line entirely, but that's okay, I think.  There's no particular reason we need to test it's an own property here, it's already tested by interfaces.html.  (The same could be said for the rest of the asserts changed in this file, but leaving them also seems harmless.)

LGTM now, just needs someone to merge it.